elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemSchema.java

```diff
@@ -53,9 +53,10 @@ public enum ElytronSubsystemSchema implements PersistentSubsystemSchema<ElytronS
     VERSION_16_0(16),
     VERSION_17_0(17),
     VERSION_18_0(18),
-    VERSION_18_0_COMMUNITY(18, Stability.COMMUNITY),
+    VERSION_19_0(19),
+    VERSION_19_0_COMMUNITY(19, Stability.COMMUNITY),
```

Comment:
WFCORE-6756 indicates the need to increment the subsystem model.  Why is the XML schema version changing?

Reply:
@pferraro we are always bumping the schema when incrementing the subsystem model, see for example previous bump jira https://issues.redhat.com/browse/WFCORE-6312 . 
So I have moved the COMMUNITY resource dynamic-ssl-context from the community_18 schema to the new schema community_19, if that is okay for this Final release? Maybe that's too late now, let me know. @fjuma Do you agree we should increment both the model and the XML schema?

Reply:
> we are always bumping the schema when incrementing the subsystem model

That is not correct.  Schema versions and model versions are not inherently related.  A new version of a subsystem schema is only required if there are changes to the XML representation of the model.  A new version of a subsystem model is only required when there are changes to the subsystem's resource description (read recursively).  The XML representation change change without the model, just as the model can change without affecting the XML representation.

> see for example previous bump jira https://issues.redhat.com/browse/WFCORE-6312

As a general note, it is not advisable to create JIRA issues for this.
Changes like WFCORE-6312 are not supposed to be merged independently from a schema and/or model change that requires it - otherwise this can easily result in redundant versions.  While redundant model versions have no signficant cost (other than being a potential source of confusion), redundant schema versions require parser registration and are otherwise a waste of heap space.  Tracking this creates a completely avoidable burden to the release coordinator, who would otherwise need to keep track of version bumps for every subsystem, and rollback changes if there were no associated subsystem changes.  Consequently, version bumps to a subsystem model and/or schema are not appropriate as a standalone pull request.

> So I have moved the COMMUNITY resource dynamic-ssl-context from the community_18 schema to the new schema community_19

Unless the XML representation of the subsystem's model has changed, there is reason to change this.

Reply:
@pferraro I see, thank you very much for explanation! That makes sense, I will remove the schema update from this PR

Reply:
On the general note I would say it is preferred that we do use a Jira issue for the model bump - where we have multiple engineers working on features in a subsystem independently we do want a coordinated approach so one does the work handling the bump(s) and then the individual features can depend on the common commits.

Yes we do need to ensure the bump only goes in as a feature goes in and generally this can happen when the first of any are merged.  Working at different stability levels is going to confuse that a bit more now so we may need to think some more on the coordination.

Reply:
+1, I agree that it's good to create a Jira issue for the model bump. Typically, what we do is get a draft PR open with the changes needed so that others working on new features can make use of the common commits. We make sure that the PR is only merged once we have approved the PRs for at least one of the features that make use of it.